zarf-config.toml

```diff
@@ -13,3 +13,7 @@ injector_arm64_shasum = '663df681deea957b0ec53538eab221691a83de8e95d86b8a29008af
 registry_image_domain = ''
 registry_image = 'library/registry'
 registry_image_tag = '2.8.2'
+
+# The image reference to use for the optional git-server Zarf deploys
+gitea_image = 'gitea/gitea'
+gitea_server_version = '1.19.3'
```

Comment:
newline (we always make sure that files have a trailing newline so things like regexes that match on `\n` will always work)